README.md

```diff
@@ -5,106 +5,66 @@ https://sojustrepairit.org/
 The goal of the SJRK is to help youth movements and social justice initiatives to become welcoming environments for youth with learning differences, and benefit from the advantages of inclusive design. The resources available in the SJRK are intended to be lightweight and easy to deploy, share, repurpose and reuse. The kit will also be openly available to any group or individual hosting youth movements, youth action events and social justice movements.
 
 ## The Storytelling Tool
-Overall documentation for this project is available at:
+More information on this project is available at:
 https://wiki.fluidproject.org/display/fluid/Web+Storytelling+Tools
 
 This repository represents the Storytelling Tool, a project which anyone can use to share their story with the world in a way that is inclusive, multimodal and accessible. The project primarily uses [Fluid Infusion](https://fluidproject.org/infusion.html), so it is assumed that anyone working on it will have some familiarity with Infusion's syntax and philosophy.
 
 This project also uses [Node.js](https://nodejs.org), [Kettle](https://github.com/fluid-project/kettle) and [CouchDB](http://couchdb.apache.org/) to build the web-hosting environment that drives the tool.
 
-## How Do I Use It?
+## How do I use it?
 
 ### Development
+The tool is split up into two main pieces: a webserver to manage communication with the CouchDB database, and an HTML interface for creating and sharing stories.
+* Server code can be found in `src/server`
+* Common JavaScript code can be found in `src/ui`, though a significant portion of the code may be present in the theme folders (see [Theme Customization](#Theme-Customization)).
+
+## Configuring the application
+While Infusion allows for just about any possible extension you can imagine and implement, there are a few handy configuration settings provided as part of the code which allow you, for instance, to customize the current theme or configure a new one. The server will load a file named `sjrk.storyTelling.server.config.json5`, and the `globalConfig` collection is where you will find the following settings:
+
+| Setting | Description |
+|---------|-------------|
+| `port`  | _(Optional)_ The HTTP port the server will be hosted on. The default value is 8081. |
```

Comment:
Looking at the source code, `globalConfig` also supports: binaryUploadDirectory, uploadedFilesHandlerPath, deletedFilesRecoveryPath, secrets.adminPass.

Since `sjrk.storyTelling.server.config.json5` is in json5 format, it might be helpful to write a comment section at the start to list all supported values and their goals, as a reminder to end users and developers.

Reply:
The three upload-related keys (`binaryUploadDirectory`, `uploadedFilesHandlerPath`, and `deletedFilesRecoveryPath`) were previously pulled in from the configs file, but after further thought I decided these are not "quick and easy" things to configure. If these values are changed, then the code needs to be updated to create and include the referenced directories, as well as to ensure they have the correct permissions set. This means it's non-trivial, and so I think if anyone is going to be digging that far into customizing the application, they'll be able to find the values in `serverSetup.js`. Is that a reasonable assumption?
I suppose I could put comments in the JS to explain their purpose, if that helps.

As for `secrets.adminPass`, this is something I believe we've discussed in the past. The admin password is something that should be kept closely guarded, and having it alongside more "open" values may be misleading or lead to it being overlooked. I suppose I could keep it at the `secrets` path within `globalConfig` to make that more obvious, but I think we decided that having it in a separate file means that the developer will make a conscious move to create that file and keep track of it separately. It's not a perfect solution in either case, but it may prevent accidental disclosure. I'm not 100% sold on this idea haha

Reply:
If these properties are not intended to be configured by end users, they probably should be moved out of `globalConfig` into component options or even member options to prevent them from being accidently overwritten or potential security issues with the present of `adminPass`.